VideoApi/VideoApi.AcceptanceTests/packages.lock.json

```diff
@@ -684,28 +531,6 @@
           "System.Threading.Tasks": "4.0.11"
         }
       },
-      "Microsoft.Azure.ServiceBus": {
-        "type": "Transitive",
-        "resolved": "5.0.0",
-        "contentHash": "kL+xqsI39R+SYnwWimKI8ODdSBJjUd+PGDbjGaALN2rTrVijBee/tvAHHiKR2g4uMQk1MCo1DcLVQmq3H+vsfA==",
-        "dependencies": {
-          "Microsoft.Azure.Amqp": "2.4.6",
-          "Microsoft.Azure.Services.AppAuthentication": "[1.0.3, 2.0.0)",
-          "Newtonsoft.Json": "10.0.3",
-          "System.Diagnostics.DiagnosticSource": "4.5.1",
-          "System.IdentityModel.Tokens.Jwt": "5.4.0"
-        }
-      },
-      "Microsoft.Azure.Services.AppAuthentication": {
```

Comment:
a lot of nuget packages appears to have been removed (although it appears its from the test project) Just to confirm that its intentional?